python/setup.py

```diff
@@ -63,6 +63,7 @@ def read(fname):
             "markdown==3.3.7",
             "pymdown-extensions",
             "mkdocs-macros-plugin==0.7.0",
+            "mkdocs-minify-plugin==0.6.4",
```

Comment:
This PR doesn't work. You need to relax the version of the minify plugin. See te tests results.